### DISCUSSION TITLE
Timer sunrise and sunset does not allow individual time settings, Tasmota 10.0.0

### DISCUSSION THREAD

**Discussion by rudi48** [arendst/Tasmota#13896] [Q&A] [answered]
I had a problem with Tasmota 8.5.0 with the Timer and sunrise and sunset individual offset time settings.
It was not possible the set an individual time offset e.g 1 hour to sunrise and sunset.
If I set an offset of e.g +1h in sunrise it will also appear (parially) in sunset or vice versa.

Then I upgraded to Tasmota 10.0.0, but with the same behaviour.
The Timer setup was done via network in the web mask.
The device is a Sonoff 230 VAC switch.
My documentation of the project:
https://www.rudiswiki.de/wiki9/ioBroker-sonoffPOWR2

The reason for the daylight switching is to reduce the standby power of 1.4 W of the solar inverter.

Is it possible that I am doing something wrong, or is it a Tasmota bug?

Regard, Rudi

**Comment by Jason2866** [comment, marked as answer]
You define a action (Timer 1) "on" with sunrise and your offset and define a action (Timer 2) "off" with sunset and your offset.

**Reply by rudi48** [reply]
Hello Jason2866,
Thank you very much for the hint, that was my fault.
I forgott to switch between timer 1 and 2.
Now it works as expected.
Regards, Rudi
